worlds/ladx/Tracker.py

```diff
@@ -236,5 +253,4 @@ async def set_item_tracker(self, item_tracker):
             if self.ws:
                 await self.send_all_inventory()
         else:
-            await self.send_inventory_diffs()
-
+            await self.send_inventory_diffs()
```

Comment:
EoF newline should be added back
```suggestion
            await self.send_inventory_diffs()
            
```